lib/Runtime/PlatformAgnostic/Intl.h

```diff
@@ -5,6 +5,52 @@
 
 #pragma once
 
+namespace PlatformAgnostic
+{
+namespace Intl
```

Comment:
I don't think there's any reason to declare these two namespaces twice in this file. Can you rearrange the file so that you have these namespaces only once. You can #ifdef INTL_ICU in more than one place if necessary.